inc/cloner/class-cloner.php

```diff
@@ -1335,9 +1341,11 @@ protected function cloneMetadata() {
 			}
 		}
 
-		// Remove the current user from the author field in Book Info
 		$user_data = get_userdata( get_current_user_id() );
-		$this->contributors->unlink( $user_data->user_nicename, $metadata_post_id );
+		if ( ! in_array( $user_data->user_nicename, $authors_slug, true ) ) {
```

Comment:
We don't want to unlink if we find the same contributor is an author in the source book (same contributor means that contains same slug or name). Test case: clone a book whose author contains the same slug or name as the one automatically generated after cloning in the target book.